custom_components/myskoda/climate.py

```diff
@@ -179,33 +190,116 @@ def __init__(self, coordinator: MySkodaDataUpdateCoordinator, vin: str) -> None:
         )
         ClimateEntity.__init__(self)
 
+        self._attr_supported_features = (
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_OFF
+        )
+
+        if self.vehicle.has_capability(
+            CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING
+        ) or self.vehicle.has_capability(
+            CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY
+        ):
+            self._attr_supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
+
+    def _has_any_capability(self, cap: list[CapabilityId]) -> bool:
+        """Check if any capabilities in the list is supported."""
+        return any(self.vehicle.has_capability(capability) for capability in cap)
+
+    def _has_capability(self, cap: list[CapabilityId]) -> bool:
+        """Check if all capabilities in the list are supported."""
+        return all(self.vehicle.has_capability(capability) for capability in cap)
+
     def _air_conditioning(self) -> AirConditioning | None:
         return self.vehicle.air_conditioning
 
     @property
-    def available(self) -> bool:
+    def _target_temperature(self) -> TargetTemperature | None:
+        """Return target temp object for auxiliary heater."""
+        if self._has_any_capability(
+            [
+                CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING,
+                CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY,
+            ]
+        ):
+            if ac := self._air_conditioning():
+                target_temperature = ac.target_temperature
+                if target_temperature is not None:
+                    return target_temperature
+        return None
```

Comment:
nit: `None` is the default return value when reaching the end of a method - no need to explicitly return it.

(same comment for other methods below)